CMakeLists.txt

```diff
@@ -88,24 +88,8 @@ endif ()
 
 # COMPILER CONFIGURATION:
 
-get_filename_component(Fortran_COMPILER_NAME ${CMAKE_Fortran_COMPILER} NAME)
```

Comment:
Detection of the fortran compiler seems so much cleaner now, it's great that you could get rid of these lines.